src/StructuredLogViewer/Controls/TracingControl.xaml.cs

```diff
@@ -783,6 +858,16 @@ private static Brush ChooseBackground(Block block)
         private void ResetZoom_Click(object sender, RoutedEventArgs e)
         {
             zoomSlider.Value = 1;
+            UpdatedGraph(scrollViewer.VerticalOffset + scrollViewer.ViewportWidth);
+        }
+
+        private void ScrollViewer_ScrollChanged(object sender, ScrollChangedEventArgs e)
+        {
+            if (e.HorizontalChange == 0)
+                return;

```

Comment:
I should add a setting to .editorconfig to always require braces around the if statement body.